### PR TITLE
Display correct action logs even plugins disabled

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue #126.
### Summary of Changes
This PR adds a method to load all actionlog plugins language file and calls it each time actions are being displayed to make sure the actions are displayed properly even if the plugins are disabled

### Testing Instructions
1. See https://github.com/joomla-projects/privacy-framework/issues/126 for issue description
2. Apply patch, confirm the issue is sorted.

### CODE REVIEW

**Comment by alikon** [comment]
should we consider to be protected ?

File: administrator/components/com_actionlogs/helpers/actionlogs.php [hunk 1]
```diff
@@ -189,4 +189,61 @@ public static function getContentTypeLink($component, $contentType, $id, $urlVar
...
+			)
+			->from('#__extensions')
+			->where('type = ' . $db->quote('plugin'))
+			->where('folder = ' . $db->quote('actionlog'))
+			->where('state IN (0,1)')
+			->order('ordering');
+		$db->setQuery($query);
+		$rows = $db->loadObjectList();
+
+		foreach ($rows as $row)
```
Comment (infograf768) [inline administrator/components/com_actionlogs/helpers/actionlogs.php]: what if we have no `$rows` ? Shall not we add a condition?

**Comment by manojLondhe** [comment]
Tested using Joomla patchtester component, this works fine for me.

**Comment by ReLater** [comment]
> should we consider to be protected ?

I would say yes, lock it. Just disabling should be possible.
Ah, I see locking has been pushed to Joomla 3.10 (I hope this feature will be merged then).

**Pull request merged**